content/events/2020-berlin/_index.md

```diff
@@ -0,0 +1,94 @@
+  country: Germany
+  city: Berlin
+  url: https://alte-muenze-berlin.de
+  venue: Alte Münze; Molkenmarkt 2, 10179 Berlin
+#map:
+sched_slug: kubernetescommunitydaybln
+social:
+  facebook: https://www.facebook.com/kcdberlin
+  twitter: https://twitter.com/kcd_berlin
+sponsors:
+- level: Initial Sponsors
+  orgs:
+  - name: Loodse
+    logo: sponsors/loodse.png
+    url: https://www.loodse.com/
+  - name: Kinvolk
+    logo: sponsors/kinvolk.png
+    url: https://kinvolk.io
+  - name: SysEleven
+    logo: sponsors/syseleven.png
+    url: https://www.syseleven.de/en/
+# sponsors:
+# - level: gold
+#   orgs:
+#   - name:
+#     logo:
+#     url:
+# - level: gold
+#   orgs:
+#   - name:
+#     logo:
+#     url:
+speakers:
+# - name: Phippy
+#   image: phippy-color.png
+#   twitter: https://twitter.com/cloudnativefdn
+#   bio: Phippy is a simple PHP app, trying to find a home in a cloud native world.
+#   homepage: https://phippy.io
+organizers:
+- name: Bill Mulligan
+  image: team/bill_mulligan.png
+  twitter: https://twitter.com/breakawaybilly
+  homepage:
+- name: Carlos Panato
+  image: team/carlos_panato.png
+  twitter: https://twitter.com/comedordexis
+  homepage: https://cpanato.dev
+- name: Carolin Kaps
+  image: team/carolin_kaps.png
+- name: Chris Kühl
+  image: team/chris_kuhl.jpg
+  twitter: https://twitter.com/blixtra
+  homepage: https://kinvolk.io
+- name: Christin Rehbein
+  image: team/christin_rehbein.png
+  twitter: https://twitter.com/SysEleven
+  homepage: https://www.syseleven.de
+- name: Kristin Wittig
+  image: team/kristin_wittig.png
+  twitter: https://twitter.com/Loodse
+  homepage: https://www.loodse.com/
+- name: Lexi Nadolski
+  image: team/lexi_nadolski.jpg
+  twitter: https://twitter.com/aleksanadolski
+  homepage: https://kinvolk.io/
+organizers_email: organizers-berlin@kubernetescommunitydays.org
+local_law_enforcement: 112
+medical_emergency: 112
+cfp_deadline:
+sponsor_deadline:
+---
+
+# **Kubernetes Community Days are coming to Berlin!**
+
+On May 26-27, the Kubernetes and cloud native community in Germany will gather at Alte Münze in Berlin. Join us for a two day technical conference loaded with exciting talks and plenty of opportunity to exchange and network. KCD Berlin are aimed at developers, operations people and all other IT professionals with an interest in cloud native technologies and are supported by the Cloud Native Computing Foundation.
+
+[Register](https://www.eventbrite.com/e/kubernetes-community-days-berlin-2020-tickets-90095641489)
+
+You have a Kubernetes story to tell? Great! Then don’t be shy and submit a proposal. Our CFP is open until March 15.
+
+[Submit CFP](https://sessionize.com/kubernetes-community-days-berlin-2020/)
+
+KCD is a community-organized and not for profit event and would not be possible without our partners. Don’t miss the opportunity to be part of and support this community event. Find our [Partnership Proposal](/img/2020-berlin/kcdberlin2020_sponsor_prospectus.pdf).
```

Comment:
Please change `Partnership Proposal` to `Partnership Prospectus`.

Reply:
done